app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,73 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :find_article, only: [:show, :update, :destroy, :edit, :create, :index]
```

Comment:
Don't needs to call find_article method for show, update and destroy.
there is no any method with edit.